lib/grpnames.gd

```diff
@@ -455,6 +457,7 @@ InstallTrueMethod( IsGroup, IsDihedralGroup );
 #P  IsQuaternionGroup( <G> )
 #A  QuaternionGenerators( <G> )
 ##
+##  <#GAPDoc Label="IsQuaternionGroup">
 ##  <ManSection>
 ##  <Prop Name="IsQuaternionGroup" Arg="G"/>
 ##  <Attr Name="QuaternionGenerators" Arg="G"/>
```

Comment:
This change and the one a bit above changes four functions from undocumented to documented  -- that's important for release notes, but not mentioned in the PR title or description (and hence easy to miss when creating release notes). Please mention this change in the PR title or description.

But in addition, I am a bit concerned about this: documenting `QuaternionGenerators` will cause people want to use it. But AFAICT we don't provide any methods for computing this attribute -- it is only set as a side effect of `IsQuaternionGroup`. Same for `DihedralGenerators`.

That said, I am not actively opposed to this change, but I am wary about the seemingly carefree approach to making it (I am sure you actually *did* spend time thinking carefully about it, it just isn't that clear from what is visibly documented in the system).